experiments/conf/config.yaml

```diff
@@ -8,10 +8,10 @@ hydra:
 video_dir: ${hydra:runtime.output_dir}/videos
 results_file: ${hydra:runtime.output_dir}/results.csv
 seed: 0
-num_episodes: 1000
-max_episode_length: 100
```

Comment:
Since max_episode_length should be environment specific, perhaps it should go under config for each environment?

Reply:
good call

Reply:
actually it looks like if I put these values in the configs for the environments, there's an error because the values get passed into the environments when we do `hydra.instantiate`. instead, let me figure out how to separate out different top-level configs for different experiments and put the values in there...

Reply:
okay I followed the example here: https://hydra.cc/docs/patterns/configuring_experiments/